scipy/stats/_stats_py.py

```diff
@@ -7632,15 +7632,14 @@ def ks_1samp(x, cdf, args=(), alternative='two-sided', method='auto'):
         An object containing attributes:
 
         statistic : float
-            KS test statistic, either D, D+ or D- (depending on the value
-            of 'alternative')
+            KS test statistic, either D+, D-, or D (the maximum of the two)
         pvalue : float
             One-tailed or two-tailed p-value.
         statistic_location: float
-            Value of `x` corresponding with the KS statistic; i.e., the 
+            Value of `x` corresponding with the KS statistic; i.e., the
             distance between the hypothesized cumulative distribution function
-            and the empirical distribution function is greatest at this
-            location.
+            and the empirical distribution function is measured at this
```

Comment:
Because "greatest" is not necessarily true. It could be the most negative difference or the most positive difference, which I don't think one would assume from the word "greatest".